CHANGELOG.md

```diff
@@ -13,6 +13,24 @@
   - `register` and `register_from_file` on class `snowflake.snowpark.udtf.UDTFRegistration`
   - `register` and `register_from_file` on class `snowflake.snowpark.stored_procedure.StoredProcedureRegistration`
   - `udf`, `udtf` and `sproc` in `snowflake.snowpark.functions`
+- Support `Column` as an input argument to a stored procedure call
+- Support `table_type` in `save_as_table`. You can now choose from three different `table_type` options: `"temporary"`, `"temp"` and `"transient"`
```

Comment:
I suggest this:

- Added support for \`table_type` in \`save_as_table`. You can now choose from three different `table_type` options: `"temporary"`, `"temp"` and `"transient"`

Reply:
Optionally, do you want to mention that leaving it blank results in a permanent table?

Reply:
@sfc-gh-sfan does leaving `table_type` as blank result in a permanent table? could you give me a little text blurb for that behavior?